DatadogSessionReplay/Sources/DatadogSessionReplay/DatadogCoreIntegration/RUMContextReceiver.swift

```diff
@@ -85,12 +87,18 @@ private extension FeatureBaggage {
     var rumContext: RUMContext? {
         guard let applicationID: String = self[RUMDependency.applicationIDKey],
               let sessionID: String = self[RUMDependency.sessionIDKey],
-              let viewID: String = self[RUMDependency.viewIDKey]
+              let viewID: String = self[RUMDependency.viewIDKey],
+              let serverTimeOffset: TimeInterval = self[RUMDependency.serverTimeOffset]
         else {
             // Current RUM session is not sampled
             return nil
         }
 
-        return RUMContext(applicationID: applicationID, sessionID: sessionID, viewID: viewID)
+        return RUMContext(
+            applicationID: applicationID,
+            sessionID: sessionID,
+            viewID: viewID,
+            serverTimeOffset: Int64(serverTimeOffset * 1_000)
+        )
     }
 }
```

Comment:
Given that now we have `TestUtilities` module in place, we can define missing + add new tests in https://github.com/DataDog/dd-sdk-ios/blob/4f36c8a23f4416237d8b71846caab747c75b1e68/session-replay/Tests/DatadogSessionReplayTests/DatadogCoreIntegration/RUMContextReceiverTests.swift#L11-L19